src/StableHashTraits.jl

```diff
@@ -159,7 +168,9 @@ properties are the same for `UseProperties`, the hash will be the same; etc...
 
 ## Implemented methods of `hash_method`
 
-- `Any`: `UseWrite()`
+- `Any`: either
+    - `UseWrite()` OR
+    - `UseTable()` for any type where `Tables.istable(type)` is true
```

Comment:
This change seems breaking, because now a table will hash differently than it would before, right? I'm not sure any projects at Beacon are using StableHashTraits w/ tables which means a breaking change would be less disruptive for us than it would be otherwise, but we should still make sure to bump the version number appropriately

Reply:
It think this is in fact technically breaking, however I am guessing the chance of this actually breaking anything is very low. 

For an object `x` that you want to hash:

1. If `x` is a namedtuple of table columns, `hash_method(::NamedTuple)` will apply, as it did before
2. If `x` is a array of named-tuple rows, `hash_method(::AbstractArray)` will apply, as it did before
3. `DataFrame` objects did not hash before: `hash_method(::Any)` applied in this case, and you got an error about `Base.write` not being implemented.

Other table types would have previously used `UseWrite`: which probably was an error unless the table type had already defined `Base.write` or `StableHashTraits.write`, so these were also probably errors. 

I do agree that this should bump the version from 0.1 to 0.2.

Reply:
Good point that it is often an error before. We could maybe check `applicable(write, x)` to see if there is a write method for `x`. Then, if there isn't, check if it is a table. That way we can be sure we are only changing an error path to a non-error path, which would be non-breaking. It makes the rule a bit more complicated but could still be worth it.